src/pages/ImportData.js

```diff
@@ -124,8 +124,25 @@ const ImportDataOverview = ({ header, snapshots, isDataset, snapshotResponses, u
         ])
       : url && div({ style: { fontSize: 16 } }, ['From: ', new URL(url).hostname]),
     div({ style: { marginTop: '1rem' } }, [
-      `The ${isDataset ? 'dataset' : 'snapshot'}(s) you just chose to import to Terra will be made available to you `,
-      'within a workspace of your choice where you can then perform analysis.',
+      Utils.cond(
+        [
+          isProtected(url, format),
+          () => [
+            '⚠️ The data you chose to import to Terra are identified as protected and require additional security settings. Please select a workspace that has an Authorization Domain and/or protected data setting.',
+            h(
+              Link,
+              {
+                style: { marginLeft: '1rem', verticalAlign: 'middle' },
+                href: 'https://support.terra.bio/hc/en-us/articles/360026775691-Overview-Managing-access-to-controlled-data-with-Authorization-Domains',
+                ...Utils.newTabLinkProps,
+              },
+              ['Learn more ', icon('pop-out', { size: 12 })]
```

Comment:
It's a good idea to include context about what you are learning more about (for folks that navigate by links). For instance,  "Learn more about protected data".